src/js/main.js

```diff
@@ -42,12 +48,38 @@ class Site {
   }
 
   initCoverVid() {
-    const _this = this;
-
-    if (_this.$covervidVideo.length) {
-      _this.$covervidVideo.coverVid(1920, 1080);
+    if (this.$covervidVideo.length) {
+      this.$covervidVideo.coverVid(1920, 1080);
     }
   }
+
+  bindStickyHeader() {
+    const that = this;
```

Comment:
Rather than `_this`? I can't remember what differences there are now with ES6

Reply:
yea just trying it on for size